### PR TITLE
Add max_amount parameter and update paypal_express_reponses table

### PR DESCRIPTION
**Pull request by daliwei**

### CODE REVIEW

File: db/ddl.sql [hunk 1]
```diff
@@ -58,7 +58,7 @@ CREATE TABLE `paypal_express_responses` (
   `kb_payment_transaction_id` varchar(255) DEFAULT NULL,
   `transaction_type` varchar(255) DEFAULT NULL,
   `payment_processor_account_id` varchar(255) DEFAULT NULL,
-  `message` varchar(255) DEFAULT NULL,
+  `message` text DEFAULT NULL,
```
Comment (pierre) [inline db/ddl.sql]: Shouldn't `schema.rb` be updated as well?

File: lib/paypal_express/api.rb [hunk 2]
```diff
@@ -352,6 +352,10 @@ def initiate_express_checkout(kb_account_id, amount, currency, properties, conte
         options = {}
         options[:return_url] = ::Killbill::Plugin::ActiveMerchant::Utils.normalized(properties_hash, :return_url)
         options[:cancel_return_url] = ::Killbill::Plugin::ActiveMerchant::Utils.normalized(properties_hash, :cancel_return_url)
+        if properties_hash[:max_amount]
```
Comment (pierre) [inline lib/paypal_express/api.rb]: For consistency, we should use [Utils.normalized](https://github.com/killbill/killbill-plugin-framework-ruby/blob/a0cfeb8d121a189249ea896e4763ba764a07b58c/lib/killbill/helpers/active_merchant/utils.rb#L34-L43) as well.

**Comment by pierre** [comment]
Could you also document the new plugin property in the [README](https://github.com/killbill/killbill-paypal-express-plugin/blob/master/README.md) (maybe in the table at the bottom)?

**Comment by daliwei** [comment]
@pierre Thanks for the comments. I have pushed new commits. Could you take a look?

**Comment by pierre** [comment]
:+1:

File: lib/paypal_express/api.rb [hunk 3]
```diff
@@ -353,6 +353,12 @@ def initiate_express_checkout(kb_account_id, amount, currency, properties, conte
         options[:return_url] = ::Killbill::Plugin::ActiveMerchant::Utils.normalized(properties_hash, :return_url)
         options[:cancel_return_url] = ::Killbill::Plugin::ActiveMerchant::Utils.normalized(properties_hash, :cancel_return_url)
 
+        properties_hash[:max_amount] = ::Killbill::Plugin::ActiveMerchant::Utils.normalized(properties_hash, :max_amount)
```
Comment (pierre) [inline lib/paypal_express/api.rb]: Nit: `properties_hash` doesn't need to be updated, we could simply have a local variable instead.

Comment (daliwei) [inline lib/paypal_express/api.rb]: Good point. Just submitted a PR on that.

**Pull request merged**